### ISSUE TITLE
Action to display error and stop execution

### ISSUE THREAD

**Issue by nriley** [talonhub/community#2018]
When something doesn't work as expected (particularly with accessibility in which there are sometimes unexpected failures/flakiness), I often need to:

- Display a message to the user (e.g. `talon.app.notify`/`actions.app.notify`) explaining what didn't work and sometimes what to do to work around the problem
- Raise an exception so that the action stops execution

I either end up re-implementing the logic or just being lazy and only doing the latter.

Does it make sense to add a community action to do this, and/or a custom exception subclass?

**Comment by nriley** [comment]
I can't eliminate the stack frame where the exception is raised entirely, but it does preserve the stack fine. Best I could figure out to do is add a comment to look up.

```
>>> actions.user.test_exception()
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "talon/scripting/actions.py", line 88, in __call__
  File "/Users/nicholas/.talon/user/talon_community/core/notify_exception.py", line 24, in test_exception
    actions.user.notify_exception("body", "title", "subtitle")
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "talon/scripting/actions.py", line 88, in __call__
  File "/Users/nicholas/.talon/user/talon_community/core/notify_exception.py", line 20, in notify_exception
    raise NotifyException(message)  # see caller above
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
.NotifyException: title: subtitle: body
```

Can talk more next week.
